code/lib/builder-webpack5/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "strict": true
+    "strict": true,
+    "skipLibCheck": true
```

Comment:
`case-sensitive-paths-webpack-plugin` has types issues with webpack5 which we want to ignore because we can't do anything about it. Reading the issues in the repo makes it seem like there might actually be some issues with it and webpack5.